specs/python_packages.table

```diff
@@ -1,18 +1,22 @@
 table_name("python_packages")
-description("Python packages installed in a system.")
+description("Python packages installed in a system. NOTE: when querying on windows, even without a users cross join, all user installed python packages will be returned")
```

Comment:
This is behavior observed on master. Due to the way the registry is queried for python package paths.
```
    // Enumerate any system installed python packages
    auto installPathKey = "HKEY_LOCAL_MACHINE\\" + kWinPythonInstallKey;
    genWinPythonPackages(installPathKey, results, logger);

    // Enumerate any user installed python packages
    installPathKey = "HKEY_USERS\\%\\" + kWinPythonInstallKey;
    genWinPythonPackages(installPathKey, results, logger);
```
https://github.com/osquery/osquery/blob/b875bf138755b5007cf93a4bec4554ba82fafa1a/osquery/tables/system/python_packages.cpp#L178-L186

Reply:
It is preserved in the current version.
master:
<img width="1283" alt="Screenshot 2025-01-07 at 1 19 28 PM" src="https://github.com/user-attachments/assets/4879714b-217a-473e-9d72-de8931c2b555" />
new version:
<img width="1318" alt="Screenshot 2025-01-07 at 12 09 46 PM" src="https://github.com/user-attachments/assets/4cd51509-8fdb-4b74-873f-5b790bb2c3e8" />